docs/fhir-guide/FHIR 101 - Practical Guide.ipynb

```diff
@@ -1743,34 +1742,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 22,
+   "execution_count": 16,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "DELETE http://fhir.kids-first.io/SearchParameter/patient-race\n",
-      "{'request_url': 'http://hapi.fhir.org/baseR4/SearchParameter',\n",
+      "{'request_url': 'http://10.10.1.191:8000/SearchParameter',\n",
       " 'response': {'base': ['Patient'],\n",
       "              'code': 'race',\n",
       "              'description': 'Search for Patients by their race',\n",
       "              'experimental': True,\n",
       "              'expression': 'Patient.extension.where(url = '\n",
-      "                            \"'http://fhir.kids-first.io/StructureDefinition/patient-race').extension.value\",\n",
-      "              'id': '4833',\n",
-      "              'meta': {'lastUpdated': '2020-01-15T22:54:16.648+00:00',\n",
+      "                            \"'http://fhir.kids-first.io/StructureDefinition/patient-race').value\",\n",
+      "              'id': '4842',\n",
+      "              'meta': {'lastUpdated': '2020-01-16T15:58:18.282+00:00',\n",
       "                       'versionId': '1'},\n",
-      "              'name': 'patient-race-search-parameter',\n",
+      "              'name': 'patient-race',\n",
       "              'publisher': 'Kids First DRC',\n",
       "              'resourceType': 'SearchParameter',\n",
       "              'status': 'active',\n",
       "              'target': ['Patient'],\n",
       "              'type': 'string',\n",
-      "              'url': 'http://fhir.kids-first.io/SearchParameter/patient-race',\n",
-      "              'xpathUsage': 'normal'},\n",
+      "              'url': 'http://fhir.kids-first.io/SearchParameter/patient-race'},\n",
       " 'status_code': 201}\n",
-      "POST http://hapi.fhir.org/baseR4/SearchParameter, patient-race-search-parameter\n"
+      "POST http://10.10.1.191:8000/SearchParameter, patient-race\n"
```

Comment:
I am leaving this comment here even though it's about the line 1776 which I can't comment on since it's not a new commit.
Per `validate(patient_race_sp, print_result=True)` the function `validate` seems not defined anywhere and based on the function naming convention, `validate_sp` seems more consistent?